### DISCUSSION TITLE
Is it possible to make data in Budibase publicly queryable with custom endpoints?

### DISCUSSION THREAD

**Discussion by gitatmax** [Budibase/budibase#1758] [Help] [answered]
I may have missed it in the docs and in the Discussions here, but I'd love if I could use Budibase to essentially manage a dataset but also make it publicly accessible through either a REST API or similar. Is this currently possible?

Thanks in advance. :)

**Comment by mike12345567** [comment, marked as answer]
Hi @gitatmax - this isn't currently a feature we support, there are REST APIs that you can use but we haven't documented them and they could be open to change.

As part of our roadmap we've laid out some plans for "Budibase as a backend" - the idea being that we provide well documented, easy to use REST APIs for getting access to data through Budibase, this will be getting worked on in the near future.

Edit: if you are interested in the internal APIs, the `koa` definitions can be found [here](https://github.com/Budibase/budibase/blob/master/packages/server/src/api/routes/row.js) - however I will warn that there is fairly major changes occurring right now in this API as part of the #1732 PR.

**Reply by gitatmax** [reply]
Hey @mike12345567, thanks for the quick reply! Much appreciated & I'll keep an eye on those links & Budibase in general :)

**Reply by mike12345567** [reply]
Just to add, over the next few weeks we'll be making the community a lot more aware of what we are actively working on/when to expect certain features, we'll be updating our roadmap which can be found here: https://github.com/Budibase/budibase/projects/10

So if you want to follow along/see when to expect certain features that'll be the place to look!

**Reply by shogunpurple** [reply]
Just a quick update - the roadmap above is now up to date with our high level feature plans for the next few months 👍
